server/utils/articles/article-search.utils.js

```diff
@@ -0,0 +1,24 @@
+import Response from '../response.util';
+
+class SearchValidators {
+/**
+ * @param {object} req
+ * @param {object} res
+ * @param {any} next
+ * @returns {any} response
+ */
+  static async checkQueryParams(req, res, next) {
+    const { author, title, tag } = req.query;
+    if (!author && !title && !tag) return Response(res, 422, 'No search parameters inputted');
```

Comment:
Please, can you explain what this validation does? I need clarification about this

Reply:
At least one of the parameters should be present. If none of them is present, the user receives an error response